docs/How_To/Tutorials/document_reader.rst

```diff
@@ -0,0 +1,253 @@
+Create a PDF Reader with Sherpa
+===============================
+
+In this tutorial, we will create a simple PDF reader using Sherpa. The PDF reader will be able to open a PDF file, load its content into a vector database and then use a question answering agent to answer questions about the content of the PDF file.
+
+
+Overview
+********
+
+To create the PDF reader, there are three main components to define:
+1. A text embedding tool to convert text and queries into vectors (We will use the SentenceTransformer library)
+2. A vector database to store the text embeddings of the PDF file (We will use Chroma in-memory vector database)
+3. An customized Sherpa Action to enable searching the vector database and can be used by an agent from Sherpa
+4. A question answering agent to answer questions about the content of the PDF file (We will use the QAAgent from Sherpa)
+
+Install dependencies
+*********************
+First, install Sherpa 
+
+.. code-block:: bash
+
+    pip install sherpa_ai
+
+
+Then, install the following dependencies:
+- pdfminer.six for extracting text from PDF files
+- sentence-transformers for text embedding
+
+.. code-block:: bash
+
+    pip install pdfminer.six sentence-transformers
```

Comment:
I need to install LangChain and loguru too, no?

Reply:
These dependencies are installed at same time as sharpa_ai is installed.